docs/src/upgrade/release-3.7.x.asciidoc

```diff
@@ -60,6 +60,54 @@ gremlin> g.union(V().has('name','vadas'),
 
 See: link:https://issues.apache.org/jira/browse/TINKERPOP-2873[TINKERPOP-2873]
 
+==== Native transaction support
```

Comment:
A few things regarding the upgrade docs:

1. I think you should title this as "TinkerGraph Transactions" because we didn't really implemented "native transactions" in TinkerPop to apply to any graph. They are transactions specific to TinkerGraph itself as a reference implementation.
2. I'm not sure what's written here is telling the right story for the Upgrade Documentation. It's sort of focused on Gremlin Server with examples that really don't have much to do with the specific change that you implemented. I think you want to explain (a) that TinkerGraph has transaction functionality now, (b) summarize its capability briefly, (c) include Console session output for how to create a transactional TinkerGraph, add a vertex, commit/rollback (d) explain when best to use it.